sql/hive/v0.12.0/src/main/scala/org/apache/spark/sql/hive/Shim12.scala

```diff
@@ -242,6 +242,14 @@ private[hive] object HiveShim {
     }
   }
 
+  // make getConvertedOI compatible between 0.12.0 and 0.13.1
```

Comment:
The comment seems not necessary, as all of the methods in this file are for the shim purpose.